app-nexus/app-nexus-htb.js

```diff
@@ -187,7 +187,7 @@ function AppNexusHtb(configs) {
             };
 
             var uspStatus = ComplianceService.usp.getConsent();
-            queryObj.us_privacy = uspStatus.consentString;
+            queryObj.us_privacy = uspStatus.uspString;
```

Comment:
Hi, 

Can you please make a PR against canary instead of master?

Thanks.

Reply:
This bidder doesn't exist in the `canary` branch for me to make those changes.  What should I do instead?

Reply:
Let's keep it in master. Master is fine.

But we need the MRA version of this adapter.  Can you please have the MRA (older) version of this adapter with the CCPA changes?

Reply:
I'm not sure I entirely follow.  Do you want me to migrate another version of this app-nexus bidder that's using the old JPT endpoint (which is the MRA version?) and have the CCPA changes applied to that?   If so, should that go to master or canary or something else?

Reply:
You are correct.  Please create a merge request to canary for that.

Reply:
Hi @ix-certification 
To follow-up on this topic, I saw that https://github.com/indexexchange/ix-library-adapters/pull/136 was opened by someone on your end?  Is this going to go through to master?  Should I make the CCPA updates that were discussed above on that PR (since the bidder in that PR is in the MRA version)?

Please confirm.

Reply:
Hi @jsnellbaker, please ignore #136, it will not be merged in (but feel free to use the files as a template if you find them useful). Please continue to make a PR to canary from your fork.

Reply:
Hi @ix-certification 
I have created https://github.com/indexexchange/ix-library-adapters/pull/139 per your request.  Please take a look at it and let me know if anything is needed.